src/components/TxSentDialog/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  Button,
+  Typography,
+  withStyles,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+} from '@material-ui/core';
+import { injectIntl } from 'react-intl';
+
+import styles from './styles';
+
+
+const TxSentDialog = ({ txid, open, onClose, intl, classes }) => (
+  <Dialog open={open} onClose={onClose}>
+    <DialogTitle>{intl.formatMessage({ id: 'transactionSentDialog.successMsg' })}</DialogTitle>
+    <DialogContent>
+      <Typography variant="body1" className={classes.bodyPrimary}>{intl.formatMessage({ id: 'transactionSentDialog.waitingMsg' })}</Typography>
+      <Typography variant="body1">{`${intl.formatMessage({ id: 'str.transactionId' })}: ${txid}`}</Typography>
```

Comment:
If you do this without defineMessages, I think it may lead to miss some strings in the future. Especially when we want to add more languages.

Reply:
Are you sure? I just read that all it does is get the language object. [see here](https://github.com/yahoo/react-intl/wiki/API#definemessages)